vue/components/ui/molecules/notification/notification.vue

```diff
@@ -32,36 +38,45 @@
     text-align: center;
     top: 100px;
     z-index: 50;
+    transition: opacity 0.125s ease-in-out;
 }
 
 .notification.fade-leave-active {
-    animation: fade-shrink-visibility 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) forwards;
+    opacity: 0;
 }
 
 .notification > .notification-container {
-    animation: fade-grow-rise 0.35s cubic-bezier(0.645, 0.045, 0.355, 1);
+    animation: fade-grow 0.125s cubic-bezier(0.645, 0.045, 0.355, 1);
     background-color: $white;
-    border-radius: 6px 6px 6px 6px;
-    box-shadow: 0px 0px 16px rgba(45, 58, 70, 0.25);
+    border-radius: 8px 8px 8px 8px;
+    box-shadow: 0 3px 5px -1px #00000033, 0 6px 10px 0 #00000024, 0 1px 18px 0 #0000001f;
     box-sizing: border-box;
     color: #1d2631;
     display: inline-flex;
-    font-size: 13px;
-    font-weight: 600;
+    font-family: "Roboto Slab", MarkPro, sans-serif;
+    font-size: 14px;
+    font-weight: 500;
     justify-content: center;
     letter-spacing: 0.3px;
-    line-height: 18px;
+    line-height: 24px;
+    line-height: 24px;
     max-height: 100%;
     max-width: 100%;
     overflow-y: auto;
-    padding: 10px 20px 10px 20px;
+    padding: 12px 24px 12px 24px;
     pointer-events: initial;
     position: relative;
     z-index: 1;
 }
 
+.notification.dark > .notification-container {
+    background-color: $dark;
+    color: $white;
+}
+
 .notification > .notification-container > .icon {
-    padding-right: 8px;
+    display: flex;

```

Comment:
Why is this needed?

Reply:
This is neede in a specific situation in which the text container will be taller than the notification icon.
With this:
<img width="224" alt="image" src="https://user-images.githubusercontent.com/24736423/86025058-adc2c300-ba25-11ea-8aaf-e893946ae2a0.png">

Without this:
<img width="224" alt="image" src="https://user-images.githubusercontent.com/24736423/86025014-a3082e00-ba25-11ea-84a7-fc5770b3f137.png">

Reply:
Can't you solve that with `vertical-align`?

Reply:
Not possible as discussed